packages/perseus/src/styles/perseus-renderer.less

```diff
@@ -541,24 +541,20 @@
     }
 }
 
-// TODO(joel) - remove?
-.perseus-tooltip {
```

Comment:
This style wasn't really used, other than to be a base later on in `.perseus-formats-tooltip`. So, I just moved this styling to where it is used.